docs/views/Scenario_comparison.rst

```diff
@@ -0,0 +1,68 @@
+.. _<view_name>-label: scenario_comparison
+
+Scenario Comparison
+-------------------
+
+*The output comparable on the "scenario comparison view" will be determine with oemof and the input/output excel file*
+
+Attributes
+^^^^^^^^^^
+**List of scenarios**
+
+**Different comparison points**
+
+* Technical
+* Financial
+* Ecological
+
+
+**Save comparison**
+
+**Export comparison**
+
+**Import scenario**
+
+Active view-components
+^^^^^^^^^^^^^^^^^^^^^^
+
+* Possible to compare two or more scenarios with each other.
```

Comment:
Here you have to list which of the view components we already described could be used in this view